images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -246,16 +246,11 @@ $browsersAndDriversList = @(
     (Get-ChromiumVersion),
     (Get-EdgeVersion),
     (Get-EdgeDriverVersion),
-    (Get-SeleniumVersion)
+    (Get-SeleniumVersion),
+    (Get-FirefoxVersion),
```

Comment:
Should we update `Get-SeleniumVersion` to include the repo name?

Reply:
The "Get-FirefoxVersion" function was updated to mention the source repository in the readme file.